openspp-project/openspp-registry
# Spp starter improvements

## **Why is this change needed?**

To add SP-MIS and Farmer Registry in spp_starter module.

## **How was the change implemented?**

Improved spp_starter module to add the configuration of SP-MIS and Farmer Registry

## **New unit tests**

Updated test cases

## **Unit tests executed by the author**

spp_starter/tests/test_spp_starter::TestSppStarter

## **How to test manually**

- Create a server or instance with a fresh database
- Go to Apps and install spp_starter
- Configure SPP Starter with every steps
- Click Done
- Check Apps Menu if all modules based on your configuration is installed.

## **Related links**
https://github.com/OpenSPP/openspp-modules/issues/506
